src/Sentry/SentryClient.cs

```diff
@@ -113,7 +113,7 @@ public void CaptureTransaction(Transaction transaction)
                 throw new ObjectDisposedException(nameof(SentryClient));
             }
 
-            if (transaction.SpanId.Equals(SentryId.Empty))
+            if (transaction.SpanId.Equals(SpanId.Empty))
```

Comment:
Random performance bug fixes in the middle? :)

Reply:
Not performance, actually a logical bug that wasn't caught because `Equals` is not type-safe.

Reply:
I meant "the performance product".
Anyway just nagging because there's a lot of different changes here but happy to get this in. Thanks!